tools/README.md

```diff
@@ -0,0 +1,75 @@
+# KTX: Tools
+
+Utilities for working with Ktx or generating code for use with LibGDX.
+
+### Why?
+
+Like the LibGDX `gdx-tools` library, this library contains tools for development rather than use in your game itself.
+
+### Guide
+
+This module contains tools that should be not be included in the core module. It can be imported as a `buildscript` 
+`classpath` dependency so its Gradle plugin can be used. The plugin adds Gradle tasks to the project in the `ktx tools` 
+group. It can be used by adding this to the `build.gradle` file:
+
+    apply plugin: "io.github.libktx.tools"
+    
+The tools can also be used from an application. In this case, import this library as a dependency of a JVM module (not 
+the `core` game module.)
```

Comment:
This is a bit confusing, I'd rephrase it to emphasize that it's a separate application:

> The KTX tools can also be used within scripts, libraries or other separate development applications. In this case, you can import this library as a regular dependency of a JVM module. However, make sure that this library is not included in your game modules (such as `core`), as it will unnecessarily increase the application size.